### ISSUE TITLE
ERC4626 Tracker should be part of release

### ISSUE THREAD

**Issue by jaglinux** [OpenZeppelin/openzeppelin-contracts#3499]
As per the discussion https://forum.openzeppelin.com/t/erc4626-vault-implementation-for-totalassets-in-base-contract/29474 and https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3497#issuecomment-1162864795 , requesting "ERC4626 Tracker" contract as part of release.

cc @Amxx

**Comment by frangio** [comment]
I don't think we should have this in the library. There are many ways in which `totalAssets` will be implemented in user code, and as far as I can tell this should not be a common one. The assets in the vault will generally be deployed somewhere, and the asset balance of the vault is meaningless in that case.
